openfl/component/aggregator/aggregator.py

```diff
@@ -76,10 +79,11 @@ def __init__(
         single_col_cert_common_name=None,
         compression_pipeline=None,
         db_store_rounds=1,
-        write_logs=False,
-        log_memory_usage=False,
-        log_metric_callback=None,
         initial_tensor_dict=None,
+        log_memory_usage=False,
```

Comment:
are we using these variables write_logs and log_metric_callback?

Reply:
Both are for backwards compatibility. They are not used.

With addition of `TensorBoard` and `ModelCheckpoint` callback, the following arguments will be removed:
* `write_logs`
* `log_metric_callback`
* `{init,best,last}_state_path`
* `log_memory_usage` (once callbacks become plan-configurable)

@payalcha @noopurintel JFYR.